### DISCUSSION TITLE
Start Id config file?

### DISCUSSION THREAD

**Discussion by Alexander-Knives** [cc-tweaked/CC-Tweaked#821] [Q&A] [answered]
Is it possible to make a "Start Id from" Config file?

The idea is to have Several temp worlds latter combine into 1 world. All with their own computers and computer ids. i would have no control over how many would be created. therefor for management i would like to change the starting Id to 1000, 2000, 3000, etc.

The use of this is very rare and limited and not worth dedicating much thought to. Therefore i have a question, is it possible to shift or change the ids of all computers in the world, i.e. shifting by 1000, therefore no computers will bug out for having the same id?

P.s. I may have posted this question before on the wrong version..... sorry.

**Comment by SquidDev** [comment, marked as answer]
I believe the easiest solution here is to create an `ids.json` file in `$world_folder/computercraft/`, which contains `{"computer": 1000}` (alternatively, copy it from an existing world and edit it).

Obviously worth noting that this isn't a good solution when one player creates 1000 computers - just worth bearing that in mind!

**Reply by Alexander-Knives** [reply]
I see... 
So if i create a .json file containing  {"computer": 1000} in the world/computercraft/ the ids will start from 1000, and count up? i.e. 1001, 1002, 1003 etc.? 

if so that will be very useful for me. 

The use case would be ~3 to 4 servers containing the same map then later combining. therefore i have no idea how many Pc/tablets/turtle would be in use. so I'm trying to future proof.

**Reply by Alexander-Knives** [reply]
Now that i've tested it. id.json is basicly logs the latest created computer
